core/common/build.gradle.kts

```diff
@@ -0,0 +1,13 @@
+plugins {
+    id("samplecomposeanroid.android.library")
+    id("samplecomposeanroid.android.hilt")
+}
+
+android {
+    namespace = "com.loodos.samplecomposeanroid.core.common"
+}
+
+dependencies {
+    implementation(libs.kotlinx.coroutines.android)
```

Comment:
coroutines için build logicte convention plugin ihtiyacı olabilir diğer modellerde kullanacaktır nowin androidde bi bakalım